### ISSUE TITLE
[Feature] Update `transformation` types for Cloudinary Video Player

### ISSUE THREAD

**Issue by colbyfayock** [cloudinary-community/cloudinary-util#202]
# **Feature Request**

## **Is your feature request related to a problem? Please describe.**

We should update the types for Cloudinary Video Player for the available options:

https://github.com/cloudinary/js-url-gen/blob/master/src/types/types.ts#L431

This is the API that's used in the underlaying Video Player, via:

https://github.com/cloudinary/cloudinary-video-player/blob/edge/src/plugins/cloudinary/common.js#L55

## **Describe the solution you'd like**

Should be able to, for the most part, copy in all of the types from the above int othe types/cloudinary-video-player.ts file

https://github.com/cloudinary-community/cloudinary-util/blob/main/packages/types/src/types/cloudinary-video-player.ts

**Comment by nooras** [comment]
Hey @colbyfayock 
i am interested in this task. Can u assign me ?
shall i copy full block of LegacyITransforamtionOptions ? if yes! My code changes are ready let me know if its correct ?

**Comment by colbyfayock** [comment]
i think we may be able to import it directly from the dependency, but import it specifically as a type, so that the dependency doesnt impact the bundle size

```
import type Transformation from ...
```

**Comment by nooras** [comment]
@colbyfayock 
Unable to import Transformation directly in cloudinary-video-player.ts
Able to import in url-loader!
What should i do? shall i add dependency in type folder ?

![image](https://github.com/user-attachments/assets/d52c08bf-2748-4550-8073-d7792958a4e1)

**Comment by colbyfayock** [comment]
yeah you can try adding the dependency, npm install `@cloudinary/url-gen`
